process/project-lifecycle.md

```diff
@@ -0,0 +1,178 @@
+
+#### Sandbox Entry Requirements and Considerations
+
+* Projects must have a minimum of two maintainers with different company affiliations.
+* Projects must be aligned with the OpenSSF mission _and_ either be a novel approach for existing areas, address an unfulfilled need, or be initial code needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+* If contributing an existing Project to the OpenSSF, the contribution must undergo license and IP due diligence by the Linux Foundation (LF).
+
+
+### Incubating
+
+Incubating projects represent maturing but not fully realized projects. Incubating projects have an established technical vision and working project, but are still refining implementation, discovering (and adapting to) new use cases, and operating as an open source team.
+
+#### Project Responsibilities
+* Bi-annual updates to the TAC on technical vision and progress on vision
+* Maintain a diversified contributor base (i.e. not a single-vendor project) with an active flow of contributions
+* Follow security best practices (as recommended by the OpenSSF and others)
+* Maintain a point of contact for vulnerability reports
+* Implement, practice, and refine mature software development and release practices such as following a version schema
+* Begin to establish the appropriate governance that enables its sustainment for potential graduation
+
+#### Project Support
+* Receives guidance on technical direction from TAC
+* Receive a TAC or WG sponsor for guidance as an Incubation project
+* Receives space for Contributor Summits at OpenSSF bi-annual events
+* Receives OpenSSF Code of Conduct Committee support
+* Receives infrastructure support (details determined by project leads and OpenSSF Budget Committee)
+* Receives support with vulnerability disclosure from the OpenSSF (Vulnerability Disclosure WG)
+* May post project updates and tutorials to the OpenSSF blog
+* May request OpenSSF budget for project improvements such as security audits or time-bound contracting needs
+* With additional TAC or WG approval, may fundraise for dedicated project funds, coordinated by the OpenSSF
+* Projects may use the OpenSSF logo to promote their project (in accordance with the trademark guidelines). Projects may not be referred to as an "OpenSSF Project" or "OpenSSF $ProjectName." Projects may say they are an "OpenSSF Incubating Project."
+
+#### Incubation Entry Requirements and Considerations
+
+* Projects must have a minimum of three maintainers with a minimum of two different company affiliations.
+* Projects should be able to show adoption by multiple parties and adoption's value to the open source community and/or end users (may include adoption of beta/early versions).
```

Comment:
Do we mean "end users" as individuals, or "end users" in the sense of consuming organizations?

Reply:
I think it is intentionally a bit vague to leave space for projects to make the case. "Open source community" covers individuals who are maintainers or OSS project contributors. I think most of the relevant projects "end users" would be organizations, but if a team showed a solution that had wide adoption by consumers (eg an IDE or browser plugin), I think the argument should be heard.

Reply:
+1 @annabellegoth2boss